### PR TITLE
chore: update obsolete function

### PR DESCRIPTION
**Pull request by thienNH0**

### CODE REVIEW

**Review by long-duc-nguyen-skymavis — CHANGES_REQUESTED** [review]
+ SHOULD: Wrap deprecated API in `#region Deprecated`.
+ MUST: Add 1 separator line for each obsolete method.

File: Runtime/Waypoint.cs [hunk 1]
```diff
@@ -90,5 +90,55 @@ private static string ExecuteWithRandomState(Action<string> callback)
...
+            WaypointSettings waypointSettings = new WaypointSettings()
+            {
+                mavisHubSessionID = sessionID,
+                mavisHubPort = port
+            };
+
+            SetUp(waypointSettings);
+        }
+
+        public static void Init(string waypointOrigin, string clientId, string deeplinkSchema, string rpcUrl, int chainId)
```
Comment (long-duc-nguyen-skymavis) [inline Runtime/Waypoint.cs:109]: MUST: Need `Obsolete` attribute as well.

File: Runtime/Waypoint.cs [hunk 2]
```diff
@@ -90,5 +90,55 @@ private static string ExecuteWithRandomState(Action<string> callback)
             callback(state);
             return state;
         }
+
+        #region Deprecated Code
```
Comment (long-duc-nguyen-skymavis) [inline Runtime/Waypoint.cs:94]: SHOULD: Call it `Deprecated API`.

File: Runtime/Waypoint.cs [hunk 3]
```diff
@@ -90,5 +90,55 @@ private static string ExecuteWithRandomState(Action<string> callback)
...
+        [Obsolete("To be removed in 0.5.0. Use SkyMavis.Waypoint.Waypoint.PersonalSign() instead.")]
+        public static string OnPersonalSign(string message, string from = null) => PersonalSign(message, from);
+
+        [Obsolete("To be removed in 0.5.0. Use SkyMavis.Waypoint.Waypoint.SignTypedData() instead.")]
+        public static string OnSignTypeData(string typedData, string from = null) => SignTypedData(typedData, from);
+
+        [Obsolete("To be removed in 0.5.0. Use SkyMavis.Waypoint.Waypoint.SendNativeToken() instead.")]
+        public static string OnSendTransaction(string receiverAddress, string value, string from = null) => SendNativeToken(receiverAddress, value, from);
+
+        [Obsolete("To be removed in 0.5.0. Use SkyMavis.Waypoint.Waypoint.SendTransaction instead.")]
```
Comment (long-duc-nguyen-skymavis) [inline Runtime/Waypoint.cs:138]: MUST: `SendTransaction()` with parentheses `().

**Review by long-duc-nguyen-skymavis — CHANGES_REQUESTED** [review]
MUST: Use `SkyMavis.Waypoint` instead of `SkyMavis.Waypoint.Waypoint`.

File: Runtime/Waypoint.cs (hunk 1 above)
Comment (thienNH0) [reply Runtime/Waypoint.cs:110]: Done !

File: Runtime/Waypoint.cs (hunk 2 above)
Comment (thienNH0) [reply Runtime/Waypoint.cs]: Done !

File: Runtime/Waypoint.cs (hunk 3 above)
Comment (thienNH0) [reply Runtime/Waypoint.cs]: Done !

**Review by long-duc-nguyen-skymavis — APPROVED** [review]

**Pull request merged**